idb/postgres/postgres.go

```diff
@@ -204,7 +204,7 @@ func (db *IndexerDb) AddBlock(vb *ledgercore.ValidatedBlock) error {
 		}
 		defer w.Close()
 
-		if block.Round() == basics.Round(0) {
+		if uint64(block.Round()) == uint64(0) {
```

Comment:
Same comment as above--the block object we're using will eventually return the proper type. Except here we can't even get rid of the `basics` import so I'm not sure how much benefit this gives us.